parent/pom.xml

```diff
@@ -392,6 +395,41 @@
         <version>${junit.version}</version>
       </dependency>
 
+      <!-- https://junit.org/junit5/docs/current/user-guide/#running-tests-ide-intellij-idea -->
+      <dependency>
```

Comment:
@JonathanGiles @srnagar 

Junit Version 5.5.2 did Not Worked so i downgraded to to the Last working Version as remarked in one of my Commits Above

„ - revert to junit 5.4.2 because of https://youtrack.jetbrains.com/issue/IDEA-223700“

Reply:
@mclay - thanks for providing the link to the issue with version 5.5.2. I am okay with using 5.4.2 for now but consider using the bom for 5.4.2.

Also, add a comment linking the above issue with 5.5.2.

```
<!-- Version 5.5.2 fails to discover tests - https://youtrack.jetbrains.com/issue/IDEA-223700 -->
<junit-jupiter.version>5.4.2</junit-jupiter.version>
```

```
<dependency>
    <groupId>org.junit</groupId>
    <artifactId>junit-bom</artifactId>
    <version>${junit-jupiter.version}</version>
    <type>pom</type>
</dependency>
```

Reply:
done